packages/plugin/src/modules/helpers/wrapper.js

```diff
@@ -128,6 +128,17 @@ export function wrap(visitor, programNode, opts) {
     ...preDefine,
     generateDefine(body, imports, exportGlobal || opts.exportAllGlobal),
   ];
+
+  // ensure copyright comments to stay leading comments of the program
+  const copyright = body?.[0]?.leadingComments?.find((comment, idx, arr) => {
+    if (comment.value.startsWith("!")) {
+      arr.splice(idx, 1);
+      return true;
+    }
+  });
+  if (copyright) {
+    programNode.body[0].leadingComments = [copyright];
```

Comment:
Would there be a risk that this overrides any existing `programNode.body[0].leadingComments`? Let's say when the first statement in the file is a simple variable declaration and this declaration has a leading comment. Would that comment be there in the previous `leadingComments` which is now overridden?

Reply:
There would be no leading comments at the program because this is being a generated block and the original body with the comments has been put inside. Therefore this copyright comment needs to be lifted to the outer program again!